src/main/java/com/koval/resolver/core/Launcher.java

```diff
@@ -219,13 +221,38 @@ public void printIssueFields() {
         LOGGER.info("Not exists available fields.");
       } else {
         LOGGER.info("Available fields:");
-        fields.forEach(field -> LOGGER.info("Field '{}' with id {}", field.getName(), field.getId()));
+        if (configuration.getConnectors().getJira().getPrintFields() == null || configuration.getConnectors().getJira().getPrintFields().isEmpty()) {
+          fields.forEach(field -> LOGGER.info("Field '{}' with id {}", field.getName(), field.getId()));
+        } else {
+          printIssueFieldsInCsv(fields, configuration.getConnectors().getJira().getPrintFields());
```

Comment:
We need to log the output information regardless of whether the parameter is set.
The code should be as follows:
`fields.forEach(field -> LOGGER.info("Field '{}' with id {}", field.getName(), field.getId()));`
`if (configuration.getConnectors().getJira().getPrintFields() != null &&`
`    !configuration.getConnectors().getJira().getPrintFields().isEmpty()) {`
`    printIssueFieldsInCsv(fields, configuration.getConnectors().getJira().getPrintFields());`
`}`